apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -8470,7 +8470,7 @@ body.update-scroll-hidden {
     }
 }
 .manage-account-popup.full-screen-modal {
-    z-index: 99999;
+    z-index: 999999;
```

Comment:
@ravinderwalkover please check this once before merging

Reply:
Done